src/matrix/room/timeline/Timeline.js

```diff
@@ -664,7 +666,7 @@ export function tests() {
                 relatedEventId: reactionEntry.id
             }}));
             await poll(() => timeline.entries.length >= 3);
-            assert.equal(messageEntry.pendingAnnotations.get("👋").count, -1);
+            assert.equal(messageEntry.pendingAnnotations.get("👋").count, 0);
```

Comment:
AFAICT, the important part of this test is that there are no relations on the message which both `0` or `-1` indicates.

The negative `pendingAnnotations` seems a bit dubious to me but that's a separate issue. Do we really want them to ever be negative?

Reply:
Yeah, the total annotation count is calculated by the remote count + the local, pending count. When removing a remote annotation, the remote count stays 1 until we receive the remote echo. During this time, the pending count is -1 so the total count is 0.